api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -34,6 +34,50 @@ const (
 	PrivateDNSZoneModeNone string = "None"
 )
 
+// UpgradeChannel - For more information see setting the AKS cluster auto-upgrade channel [https://docs.microsoft.com/azure/aks/upgrade-cluster#set-auto-upgrade-channel].
```

Comment:
```suggestion
// UpgradeChannel is ...
// For more information see setting the AKS cluster auto-upgrade channel [https://docs.microsoft.com/azure/aks/upgrade-cluster#set-auto-upgrade-channel].
```
Could you add a brief summary of what it is and leave the link at the end?